Add "The Roundtable" news and wiki articles (#7616)
ppy/osu-wiki: wiki/Tournaments/The_Roundtable/16_player_battle_royale/en.md

---
tags:

>   - The Round table

Reviewer: ```suggestion
  - TRT
```
this is the only tag you need in all of the articles. tags that have all their text contained in the title can be omitted

---

# The Roundtable 16 player battle royale

![The Roundtable banner](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)

> [**The Roundtable**](/wiki/Tournaments/The_Roundtable)'s **16 player battle royale** takes place the first day of the event.

Reviewer: this intro paragraph of each tournament should explain what it actually is / how it's played

if you wanna get this PR in quickly cuz news post tho, this can be addressed in a later update

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge |

## Participants

| User |
| :-- |

> | ![][flag_AU] [sakamata1](https://osu.ppy.sh/users/7562902) |

Reviewer: ```suggestion
| ::{ flag=AU }:: [sakamata1](https://osu.ppy.sh/users/7562902) |
```
should write country flags like this now, and you don't need the references if done this way

| ![][flag_US] [Kariyu](https://osu.ppy.sh/users/4733121) |
| ![][flag_CA] [RyuK](https://osu.ppy.sh/users/6304246) |
| ![][flag_US] [Trail Mix](https://osu.ppy.sh/users/3328742) |
| ![][flag_US] [im a fancy lad](https://osu.ppy.sh/users/4908650) |
| ![][flag_US] [fieryrage](https://osu.ppy.sh/users/3533958) |
| ![][flag_US] [Utami](https://osu.ppy.sh/users/7512553) |
| ![][flag_US] [Gurbzy](https://osu.ppy.sh/users/12970596) |
| ![][flag_US] [Apraxia](https://osu.ppy.sh/users/4194445) |
| ![][flag_US] [headstack](https://osu.ppy.sh/users/10322186) |
| ![][flag_US] [Mlaw](https://osu.ppy.sh/users/3126596) |
| ![][flag_US] [kablaze](https://osu.ppy.sh/users/3043603) |
| ![][flag_US] [Sytho](https://osu.ppy.sh/users/4175698) |
| ![][flag_US] [KoaLeahq](https://osu.ppy.sh/users/4321876) |
| ![][flag_US] [Mathyu](https://osu.ppy.sh/users/6303313) |

> | ![][flag_CA] [Rupertion](https://osu.ppy.sh/users/5774823) |

Reviewer: these can be lists rather than tables

## Ruleset

### General rules

- This tournament is a 16 player osu! battle royale style tournament.
- Beatmap scoring will be based on Score V2.

### Battle royale rules

The battle royale will proceed as follows:

- 16 players will be randomized into two groups of 8.
- Each group will play through 7 maps. After each map, the lowest scoring user will be eliminated.
- After both groups are narrowed down to one player each, the final players will compete in a BO13 1v1 match.

### 1v1 rules

- When both players are in the room and ready, the match will start.

> W- hen the match stats, both users will `!roll` to determine the pick and ban order. The roll winner will choose either the pick order or the ban order, and roll loser will choose the remaining order. Bans will be done in ABBA order.

Reviewer 2: - When the match starts,

Reviewer 3: @Blooshing please use the suggestion button to provide suggestion for changes

![Safari 2022-07-08 at 05 28 54](https://user-images.githubusercontent.com/191335/177923295-5c4da8e5-90ce-4a22-993b-3914082b2348.png)

```suggestion
- When the match stats, both users will `!roll` to determine the pick and ban order. The roll winner will choose either the pick order or the ban order, and roll loser will choose the remaining order. Bans will be done in ABBA order.
```
